cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/RestUtil.java

```diff
@@ -106,12 +104,8 @@ public RestTemplate createRestTemplate(HttpProxyConfiguration httpProxyConfigura
     }
 
     private List<HttpMessageConverter<?>> getHttpMessageConverters() {
-        List<HttpMessageConverter<?>> messageConverters = new ArrayList<HttpMessageConverter<?>>();
-        messageConverters.add(new ByteArrayHttpMessageConverter());
-        messageConverters.add(new StringHttpMessageConverter());
-        messageConverters.add(new ResourceHttpMessageConverter());
-        messageConverters.add(new MappingJackson2HttpMessageConverter());
-        messageConverters.add(new LoggregatorHttpMessageConverter());
-        return messageConverters;
+        return Arrays.asList(new ByteArrayHttpMessageConverter(), new StringHttpMessageConverter(),
```

Comment:
The previous was more readable.

Reply:
+1

Reply:
You mean the each class on a new line or just the list object creation and addition in general?

Reply:
You will not be able to format it each argument on a new line, because it contradicts with our formatter.
So the only option is the old way.

Reply:
Each class on a new line, but since you can't force the formatter to format the current statement this way, just revert back to the list object creation.